backend/pkg/api/dashboard.go

```diff
@@ -143,27 +143,29 @@ func NewDashboardResponse(response *http.Response) (*DashboardResponse, error) {
 	defer response.Body.Close()
 
 	body, err := ioutil.ReadAll(response.Body)
-
 	if err != nil {
-		return &dashboardResponse, err
+		log.DefaultLogger.Error("NewDashboardResponse: ioutil.ReadAll: " + err.Error())
```

Comment:
Just curious because of the `DefaultLogger`, is there a way to define a custom logger? Maybe one which allowed you to do something like:

```
log.EmailerLogger.Error(LOG.NEW_DASHBOARD_RESPONSE.IO_UTIL_READ_ALL, err)
```

...or something a bit better!

Reply:
I haven't tried to create a custom logger - i'm sure there's probably a way